### ISSUE TITLE
[DX] Shorten the way to try out Nuxt

### ISSUE THREAD

**Issue by Baroshem** [nuxt/nuxt#19407]
Hey!

I noticed something in Nuxt docs about trying it out / quickstart / getting started. It is relatevely a long way for the user that just types nuxt.com and opens a website to actually go through the docs and see the actual nuxi command to try it out.

1. Access nuxt.com website
2. Click Get Started
3. Read whole page about Introduction
4. Navigate manually to Installation
5. Scroll to https://nuxt.com/docs/getting-started/installation#new-project
6. Actually type the nuxi command to generate a new project

From my DX experience this path is a bit too long and hard to follow. It could be easily fixed by following a pattern we leverage in major modules where a command to install the module is actually on the homepage:

![image](https://user-images.githubusercontent.com/37120330/222641423-0280bd18-197e-4440-babe-78be05458859.jpeg)

**My proposition:**

I would recommend to have a simple bash terminal in the Nuxt.com website just below the H1 and the diamond, with a simple nuxi new command that would allow users first seeing the nuxt.com page to easily generate a new project without searching for the actual command further in the docs.

This should decrease the number of steps and the time new users need to spend in order to try out Nuxt which should eventually help improving DX :)

Let me know what you think about it

**Comment by Baroshem** [comment]
Also, if you agree, I could work on adding this to the docs :)

**Comment by harlan-zw** [comment]
Agreed, good idea!

**Comment by Baroshem** [comment]
This is even better idea @harlan-zw !

If the nuxt.com is not OS yet, they you are right, I cannot add it. So I am leaving this up to you  and @Atinux to work with :)

**Comment by Atinux** [comment]
Made a small update to start that the Get started button now goes to the installation page.

**Comment by Baroshem** [comment]
@Atinux What about including the terminal on the homepage as Harlan suggested? I think it can drastically help users to faster try Nuxt out 🙂

**Comment by Atinux** [comment]
I know I tried but right now I am not happy about the result (also have a small bug):

https://nuxt.com/?preview=422205636c782eac46bd5691e584662bc48064027b82e252fc97c493756e03fb

**Comment by Atinux** [comment]
It’s live ❤️

![F9D814D4-EE74-477F-877A-A035B7ED8ADE.png](https://volta.s3.fr-par.scw.cloud/F9_D814_D4_EE_74_477_F_877_A_A035_B7_ED_8_ADE_3bebdc73e1.png)
